test/fixtures/test/roles.json

```diff
@@ -61,7 +61,8 @@
       "work:manage",
       "work:owned:manage",
       "app:schedule:reduced",
-      "work:own"
+      "work:own",
+      "dashboards:view"
```

Comment:
In the `reduced-schedule => find in list` test, I need to verify that the FIL query persists on a route change. 

Adding the `"dashboards:view"` permission adds the `Dashboards` link to the app nav for reduced schedule users. Which allows me to easily go to the `/dashboards` page and then come back to the schedule to verify the FIL query persisted.